pallets/stake/src/lib.rs

```diff
@@ -111,17 +111,17 @@ impl<A, B: HasCompact> Into<IndividualExposure<A, B>> for Bond<A, B> {
 
 #[derive(Copy, Clone, PartialEq, Eq, Encode, Decode, RuntimeDebug)]
 /// The activity status of the validator
-pub enum ValidatorStatus<BlockNumber> {
+pub enum ValidatorStatus {
```

Comment:
`ValidatorStatus` used to be generic over the inner variant of `Leaving`, but it was only ever instantiated with the `RoundIndex` type. So, the generic was removed and the inner variant of `ValidatorStatus::Leaving` was defined to always be `RoundIndex`. Earlier in this PR, something similar was done to remove RoundIndex as a generic parameter  from `CandidateState`.